pages/_app.js

```diff
@@ -18,9 +18,7 @@ function MyApp({ Component, pageProps }) {
   useEffect(() => {
     if (typeof window !== "undefined") {
       const lang = window.localStorage.getItem("lang");
-      if (lang === "en" || lang === "fr") {
-        router.push(router.asPath, {}, { locale: lang });
-      } else {
+      if (lang === null) {
```

Comment:
Perfect thank you Ephraim, I would just use a falsy condition here like `!lang` and it will cover all falsy cases